### PR TITLE
Add default_cros_reference config option

### PR DESCRIPTION
**Pull request by 4n12i**
Add the `default_cros_reference` config option to sync cros without the `--reference` option

### CODE REVIEW

File: src/repo.rs [hunk 1]
```diff
@@ -243,3 +253,17 @@ fn find_cros_dir_from_cwd() -> Result<String> {
...
+#[cfg(test)]
+mod tests {
+    use super::*;
+    #[test]
+    fn reference_match() {
+        let if_some = Some("cros".to_string());
+        let if_none = None;
+        let default = Config::read().unwrap().default_cros_reference();
+
+        assert_eq!(if_some, get_reference_repo(&if_some).unwrap());
```
Comment (hikalium) [inline src/repo.rs:266]: Please use https://doc.rust-lang.org/std/assert_matches/macro.assert_matches.html

Comment (4n12i) [reply src/repo.rs]: Thank you for letting me know!

**Review by hikalium — CHANGES_REQUESTED** [review]
please check the comment above

File: Cargo.toml [hunk 2]
```diff
@@ -39,3 +39,4 @@ whoami = "1.4.1"
 strum = "0.25.0"
 strum_macros = "0.25"
 signal-hook = "0.3.x"
+assert_matches = "1.5.0"
```
[the remark this answers is not in the document]
Comment (4n12i) [reply Cargo.toml]: Yes, we don't need it. I removed it!

[the remark this answers is not in the document]
Comment (hikalium) [reply Cargo.toml]: Nice!

**Review by hikalium — APPROVED** [review]
LGTM!

**Pull request merged**